source/openapi3/split.yaml

```diff
@@ -6313,7 +6360,7 @@ components:
               example: '2021-06-01T02:10:56Z'
       example:
         data:
-          ref: T.87xp
+          ref: 52986d9d-4788-4c41-92a1-9d364c6545e5
```

Comment:
Is this intentional? Normally `ref`s are not UUID.

Reply:
`ref` had a format of `uuid` in the schema, so I changed the example to be a uuid. I'll change the example back and remove the `uuid` format requirement.